src/test/java/spoon/test/module/TestModule.java

```diff
@@ -313,7 +314,7 @@ public void testModuleNames() {
 		batchCompiler.configure(args);
 
 		CompilationUnit[] cu = batchCompiler.getCompilationUnits();
-		List list = new ArrayList<>(Arrays.asList(new String[]{String.copyValueOf(cu[0].module),String.copyValueOf(cu[1].module)}));
+		List<String> list = Arrays.stream(cu).map(CompilationUnit::getModuleName).map(String::copyValueOf).collect(Collectors.toList());
 		assertTrue(list.contains("foo"));
 		assertTrue(list.contains("bar"));
```

Comment:
While you're poking about here anyway, how about improving it just a little bit more? This test is a bit weak (because it leaves the possibility of there being other module names in the list), and it also isn't entirely clear about its purpose.

I propose we assert on the entire list. And instead of a list use a set, because we care only about distinct module names.

```suggestion
		Set<String> moduleNames = Arrays.stream(cu)
				.map(CompilationUnit::getModuleName)
				.map(String::valueOf)
				.collect(Collectors.toSet());
		assertThat(moduleNames, equalTo(Set.of("bar", "foo")));
```

Reply:
done. I replaced the suggested deprecated assertThat with a non-deprecated one.

Reply:
Hmm, I didn't suggest the deprecated `assertThat`, I just didn't specify which one to use (in fact I didn't even remember that JUnit 4 had a built-in `assertThat`). You've imported both, so I suspect your IDE auto-imported a bunch of stuff. See review comments.